gopro_webcam.service

```diff
@@ -4,9 +4,9 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
-ExecStart=/usr/local/sbin/gopro webcam -p enx -a -n
```

Comment:
Could you elaborate on the reason you dropped this `-p enx` ?

Reply:
For my GoPro, the pattern is actually `enp` so I think specifying `-p enx` may cause some GoPros to not be detected.

Reply:
Lets revert this to the original since its a per-user choice. If we change this we'd also need to change the README

Reply:
oh, ok.. then lets keep it that way

Reply:
https://github.com/jschmid1/gopro_as_webcam_on_linux/commit/325281b22c2f038470ca6651dcda79ffee1895a4

Reply:
@jschmid1 do you think it is necessary to put the actual service file in the README? I think we can just direct users to the actual file itself.